.travis.yml

```diff
@@ -1,25 +1,25 @@
 language: node_js
-
 node_js:
   - 8
   - 6
-
 cache:
   directories:
     - ~/.npm
-
 notifications:
   email:
     recipients:
       - arizzitano@edx.org
       - jbradley@edx.org
     on_success: always
     on_failure: always
-
+  hipchat:
+    rooms:
+      secure: rb3di/8yPbXwyNvUbbeXkYkBcWhk8E2MdtLcMR3EAsBWgMB+2S13ScAT5lR2U2UuTwwZHxzdz6IoKZcTzzT13KltYBv0f+7yaga8XjBAJy5SVrMsdSRZpGHiv+6AVr9PCio/vOS3AHn3kcQA4l5HTFx5YSHuKYedzXuROb6XO6uGjsYpbHxpxvNPizPNgTL4NcJVxNBxs3DL4b89ibgliNe7zECSEzpWqwaHJG2+GE5HebS05dl5i2cEJNzC9hEse6jcv3P2+UOr3OsBMfhNc/68igp/crZbvxY5V7IjDYUYbEKdTOJT/ltaNT6NC/x0tp+RhJxyOAuBDrdzU1DYEBo23CmpbQX5s73FwPsgxyrL6PJbI7gmB6hziD45U7Cl9ABetIPJWIQlUy6Atw+1ItRHiyqxJdz9ahfKRFCGrBgV9ML5NitvHmxRjrAKG4VsP4n8Km2WbNO4KTV6xoqPf1dtXE5W3JzIBfoR5FfeqvF4mQAe7K5wZEt+cIq/Ca71cEkKZRtl9P5v6kiGjUA5RZmQIBwHvWUMnjDPsXTKVfc12rvYhsdbbdUGIxgVZ0KmyHAFvfWN9RuLc0Gu7N/60aj0wUMMmB0gva27Cb2PeeG6Af1Z4AVHnUcwwMGz8yiei3WqeWJHd3phdzBSvwHX2psUKFGkxyCDYZej06VTK3w=
```

Comment:
Used the recommended command `travis encrypt api_token@room_id_or_name --add notifications.hipchat.rooms` from the Travis docs.

Reply:
Used the `Paragon` room ID - we can add more rooms in the future, but wanted to try out `Paragon` notifications for starters.